kt/plugins/godot-gradle-plugin/src/functionalTest/kotlin/godotruntime/GodotDownloadTaskConfigTest.kt

```diff
@@ -0,0 +1,159 @@
+package godotruntime
+
+import godot.gradle.tasks.godotruntime.OS
+import org.gradle.testkit.runner.GradleRunner
+import org.jetbrains.kotlin.ir.backend.js.ic.newFile
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.io.TempDir
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.EnumSource
+import org.junit.jupiter.params.provider.ValueSource
+import java.io.File
+import java.nio.file.Path
+import java.util.*
+import kotlin.io.path.absolutePathString
+
+
+class GodotDownloadTaskConfigTest {
+
+    @TempDir
+    var testProjectDir: Path? = null
+
+    private lateinit var gradleRunner: GradleRunner
+    private lateinit var buildFile: File
+
+    @BeforeEach
+    fun setup() {
+        val root = File(testProjectDir!!.absolutePathString())
+        println(root.absolutePath)
```

Comment:
I would either remove such logs or print them "properly" with a prefix of some sort.
So when we need to look at the logs we know from where such prints are coming from and why they are printed.

Reply:
There are other such cases in this PR. Hence i won't add a review comment on each one :-)